README.md

```diff
@@ -15,18 +15,20 @@ yarn add --dev @swc/core @swc/register
 ```
 
 ### Usage
-```bash
-require("@swc/register");
-```
-
-All subsequent files required by node will be transformed by swc. You can also
-call @swc/register directly from command line.
 
-```bash
-swc-node <filename>
-node -r '@swc/register' <filename>
+```js
+require("@swc/register")({
+  extensions: [],
+  ignore: [],
+  only: [],
+  // And swc configuration
+  jsc: {},
+  env: {}
+});
 ```
```

Comment:
As the automatic hook is being removed in #14, `@swc/register` always requires an option. It is to match the behavior of `@babel/register`.